superset-frontend/src/SqlLab/components/ColumnElement.tsx

```diff
@@ -18,13 +18,47 @@
  */
 import React from 'react';
 import PropTypes from 'prop-types';
+import { ClassNames } from '@emotion/core';
+import { styled, useTheme } from '@superset-ui/core';
 
-import { OverlayTrigger, Tooltip } from 'react-bootstrap';
+import { Tooltip } from 'src/common/components/Tooltip';
 
 const propTypes = {
   column: PropTypes.object.isRequired,
 };
 
+const StyledTooltip = props => {
+  const theme = useTheme();
+  return (
+    <ClassNames>
```

Comment:
We can't simply wrap Tooltip in `styled` - here's an explanation https://stackoverflow.com/a/63500556/6881179.
I used Emotion's `ClassNames` instead of mounting a tooltip as a child of trigger node, because I didn't like the idea of Tooltip being a child of span. This feels cleaner.

Reply:
Cool, hadn't used that one before! Pretty nifty!